src/components/sky-nav/sky-nav.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import template from './sky-nav.twig';
+// https://timber.github.io/docs/guides/menus/
+const menu = {
+  items: [
+    {
+      link: '/does',
+      title: 'What We Do',
+      current: true,
+    },
+    {
+      link: '/believes',
+      title: 'Our Approach',
+    },
+    {
+      link: '/made',
+      title: 'Our Work',
+    },
+    {
+      link: '/thinks',
+      title: 'Articles',
+    },
+    {
+      link: '/presents',
+      title: 'Speaking',
+    },
+    {
+      link: '/contributes',
+      title: 'Labs',
+    },
+    {
+      link: '/is',
+      title: 'Team',
+    },
+  ],
+};
+
+<Meta title="Components/Sky Nav" />
+
+# Sky Nav
+
+This will eventually be our primary navigation component. Right now, we're just critiquing some markup updates in terms of their accessibility.
+
+<Preview>
```

Comment:
I wonder if it's worth outlining the data structure expected by this menu? Especially the `is_home`/`item.current` stuff? Or is that overkill and future devs should figure it out by reviewing the code?

Reply:
We definitely should! I left a link in the story next to the object, but we should definitely do more.